### ISSUE TITLE
Struct parameter in instance_create_* should show list of variable definitions

### ISSUE THREAD

**Issue by tristannottelman** [YoYoGames/GameMaker-Feature-Requests#1277]
### Is your feature request related to a problem?

When I intend to create a new instance using `instance_create_*` and transmit data through the struct parameter, I need to refer back to the object to ascertain the variable definitions. This is a bit frustrating and can lead to unnecessary mistakes.

### Describe the solution you'd like

When I input a variable that isn't covered by the variable definitions, a helpful warning is displayed. However, it would be even more beneficial if GameMaker could also notify me about omitted variables or provide automatic suggestions for the missing variables within the struct.

**Comment by AtlaStar** [comment]
copy of #1218, although your description _is_ better than mine

**Comment by gnysek** [comment]
Why should GM inform about omitted variables? Maybe for some you don't ever want to override them. Also, things like x,y,direction, sprite_index, image_xscale are also object properties - should those then be treated as committed?

**Comment by tristannottelman** [comment]
That's a good point. In my opinion, omitted variables should be marked as a warning by Feather, but variables from variable definitions only. I want to create generic objects that I can reuse and pass different variables. It would be nice if I can see that I am missing any variable.

**Comment by rwkay** [comment]
I am not 100% convinced on this as the struct is not restricted to only having object variables or instance variables... you can have anything in there... perhaps an opt-in preference for the intellisense

**Comment by tristannottelman** [comment]
Yes you are right. Showing hint list of variables in variable definitions should be sufficient.
